api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -2503,27 +2501,23 @@
     /**
      * Reset the {@code executionMap} field to {@code null}
      */
-    public void flushExecutionMap()
-    {
+    public void flushExecutionMap() {
         this.executionMap = null;
     }
+
     /**
      * @return a Map of executions field with {@code PluginExecution#getId()} as key
      * @see PluginExecution#getId()
      */
-    public java.util.Map<String, PluginExecution> getExecutionsAsMap()
-    {
-        if ( executionMap == null )
-        {
+    public java.util.Map<String, PluginExecution> getExecutionsAsMap() {
```

Comment:
Why java.util.Map instead of just Map?

Reply:
Using classes without the full package may lead to compilation errors if the generator does not add the related imports.  Removing the package would tie the code block to the way the generator works.  For example, it can be done for the `java.util.Map` but not for the `java.util.Iterator` below because that one is not imported.

Reply:
I would expect the code generator to figure out what imports it needs and do the right thing. If it doesn't, I guess that's another reason not to use a code generator.

Reply:
Fixed